compose/.apps/dozzle/dozzle.aarch64.yml

```
@@ -0,0 +1,3 @@
+services:
+  dozzle:
+    image: amir20/dozzle:latest
```

We can leave off `:latest`
```suggestion
    image: amir20/dozzle
```

---

81b0cf8